tests/selenium/wdio.conf.js

```diff
@@ -60,19 +63,21 @@ exports.config = {
 	// Define the different browser configurations to use ("capabilities") here.
 	// ============
 	maxInstances: 1,
-	capabilities: [ {
-		// For Chrome/Chromium https://sites.google.com/a/chromium.org/chromedriver/capabilities
-		browserName: 'chrome',
-		'goog:chromeOptions': {
-			// If DISPLAY is set, assume developer asked non-headless or CI with Xvfb.
-			// Otherwise, use --headless.
-			args: [
-				...( process.env.DISPLAY ? [] : [ '--headless' ] ),
-				// Chrome sandbox does not work in Docker
-				...( fs.existsSync( '/.dockerenv' ) ? [ '--no-sandbox' ] : [] )
-			]
+	capabilities: [
+		{
+			// For Chrome/Chromium https://sites.google.com/a/chromium.org/chromedriver/capabilities
```

Comment:
Same there should be no changes here too...is this the work of an autoformatter?

Reply:
Yes @und3fined-v01d, I had made some changes to try to run webdriver test, however, after reverting the changes the autoformater formatted the file.